sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts

```diff
@@ -717,6 +740,95 @@ describe(
       );
     });
 
+    it("Should import disbursement receipt file and send empty file content notification when the file does not contain match any records in SIMS disbursement receipts.", async () => {
+      // Arrange
+      mockDownloadFiles(sftpClientMock, [NON_MATCHING_RECORDS_FILE]);
+      // Queued job.
+      const mockedJob = mockBullJob<void>();
+
+      // Act
+      const result = await processor.processQueue(mockedJob.job);
+
+      // Assert
+      expect(result).toStrictEqual([
+        "Completed disbursement receipts integration.",
+      ]);
+      const downloadedFile = path.join(
+        process.env.ESDC_RESPONSE_FOLDER,
+        NON_MATCHING_RECORDS_FILE,
+      );
+      expect(
+        mockedJob.containLogMessages([
+          `Processing file ${downloadedFile}.`,
+          `Document number ${SHARED_DOCUMENT_NUMBER} at line 2 not found in SIMS.`,
+          `Document number ${SHARED_DOCUMENT_NUMBER} at line 3 not found in SIMS.`,
+          `Document number ${SHARED_DOCUMENT_NUMBER} at line 4 not found in SIMS.`,
+          `Processing file ${downloadedFile} completed.`,
```

Comment:
Please also assert the message `Provincial daily disbursement CSV report generated.`